docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -1,25 +1,24 @@
 .. _tutorial_create_expectations_between_tables:
 
-Create Expectations Between Tables
-===================================
+How to create Expectations between tables
+==========================================
 
-This tutorial covers the workflow of creating and editing expectations that relate to data stored in different tables or datasets.
+This tutorial covers the workflow of creating and editing :ref:`Expectations`_ that relate to data stored in different tables or datasets.
 
 Unfortunately, no Expectations natively support cross-table comparisons today (but we hope to in the future!). Consequently, there are two available paths:
 
-1. Use "Check Assets", where you create a new table that joins the two tables; or
+1. Use "check assets", where you create a new table that joins the two tables; or
 2. Use :ref:`Evaluation Parameters`_ to supply relevant metrics to expectations.
 
-Check Asset Pattern
+Check asset Pattern
```

Comment:
Lowercase "pattern"

Reply:
This one is already fixed (diff shows already lowercase).